tgui/packages/tgui/interfaces/Who.jsx

```diff
@@ -13,18 +13,22 @@ import { Window } from '../layouts';
 export const Who = (props, context) => {
   const { act, data } = useBackend(context);
   const {
-    admin,
-    all_clients,
-    total_players = [],
-    additional_info = [],
-    factions = [],
-    xenomorphs = [],
+    base_data,
+    player_additional,
+    player_stealthed_additional,
+    factions_additional,
   } = data;
 
+  const total_players = mergeArrays(
+    base_data?.total_players,
+    player_additional?.total_players,
+    player_stealthed_additional?.total_players,
+  );
+
   const [searchQuery, setSearchQuery] = useLocalState('searchQuery', '');
```

Comment:
useLocalState is depreciated. Use `useState` instead. https://react.dev/reference/react/useState